lib/00_core.sh

```diff
@@ -131,8 +131,8 @@ _PACMAN_detect() {
   [ -x "/sbin/apk" ] && _PACMAN="apk" && return
   [ -x "/bin/opkg" ] && _PACMAN="opkg" && return
   [ -x "/usr/bin/tazpkg" ] && _PACMAN="tazpkg" && return
-  [ -x "/usr/bin/swupd" ] && _PACMAN="swupd" && return
-
+  
+  command -v swupd >/dev/null && _PACMAN="swupd" && return
```

Comment:
Hi @mondeja , may I ask why we need to change this? There are some hard-coded paths there because those are quite distro-specific and the distro tends to keep them.

Reply:
It seems that my local Docker doesn't play very well with symbolic links because doesn't recognize `/usr/bin/swupd` as an executable 🤔 I've reverted it.

Reply:
@mondeja I see, that may be related to the back-end storage driver used by Docker. You can see that from `docker info` output.